src/InMemoryRepository.php

```diff
@@ -81,7 +81,7 @@ public function find($query, $language = 'glob')
         $resources = array();
 
         if (Glob::isDynamic($query)) {
-            $resources = $this->getGlobIterator($query);
+            $resources = iterator_to_array($this->getGlobIterator($query));
```

Comment:
No, ArrayResourceCollection just requires any `Traversable` object - you can see that that object is passed to `replace()`, which in turn calls `iterator_to_array()`.